src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -0,0 +1,161 @@
+package no.nav.syfo.client.pdl
+
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import no.nav.syfo.application.cache.RedisStore
+import no.nav.syfo.client.azuread.AzureAdClient
+import no.nav.syfo.client.azuread.AzureAdToken
+import no.nav.syfo.client.httpClientDefault
+import no.nav.syfo.client.pdl.domain.*
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.util.*
+import org.slf4j.LoggerFactory
+
+class PdlClient(
+    private val azureAdClient: AzureAdClient,
+    private val pdlClientId: String,
+    private val pdlBaseUrl: String,
+    private val redisStore: RedisStore,
+) {
+    private val httpClient = httpClientDefault()
+
+    suspend fun personIdentNavnMap(
+        callId: String,
+        personIdentList: List<PersonIdent>,
+    ): Map<String, String> {
+        val cachedPersonIdentNameMap = getCachedPersonidentNameMap(
+            personIdentNumberList = personIdentList,
+        )
+
+        val notCachedPersonIdentNumberList = personIdentList.filterNot { personIdentNumber ->
```

Comment:
I caching er det vanlig å kalle en bom i cachen for en miss. Hva med å navne variabelen PersonIdentCacheMisses elns?